Cargo.toml

```diff
@@ -15,7 +35,41 @@ rocksdb = "0.15"
 percent-encoding = "2"
 regex = "1"
 url = "2"
-crossbeam = "0.7"
+crossbeam = "0.8"
+
+[[bin]]
+name = "yago4"
+path = "src/main.rs"  
+edition = "2018"
+
+[profile.dev]
+opt-level=0
+debug = true
+rpath = false
+lto = false
+debug-assertions = true
+codegen-units = 4
 
 [profile.release]
-lto = true
\ No newline at end of file
+opt-level=3
+debug = false
+rpath = false
+lto = true
+debug-assertions = false
+codegen-units = 1
+
+[profile.test]
+opt-level = 1
+debug = true
+rpath = false
+lto = false
+debug-assertions = true
+codegen-units = 4
+
+[profile.bench]
+opt-level = 3
+debug = false
+rpath = false
+lto = true
+debug-assertions = false
+codegen-units = 1
```

Comment:
Why not keep using the rustc default with maybe some slight changes (lto on on `release`?)

Reply:
For sure =] Just good intentions to have the links going for docs.rs etc.

Reply:
Thank you for the invite!

Reply:
Removed myself from the crate ownership =]

Reply:
> Why not keep using the rustc default with maybe some slight changes (lto on on `release`?)

I'm somehow a person that keeps even "default" values fixed in the files (like with the rustfmt.toml) because Defaults can certainly change (with updates to rustc or cargo for example) and fixing the options I use as standards to build with gives me certainty that the environment was not changed from these values. :-P But for sure, tell me what you think should be there and I correct it. ;-)

Reply:
> Thank you for the invite!

Because I didn't push the source code (because of the failure crate still being in it and the badge saying insecure) the docs on docs.rs weren't build. So it might be needed to increase the patch version and publish another version either with `cargo publish` or with `cargo release` (that takes cargo of the changelog and the version in different places (for now the only the Cargo.toml)) just to have the crate also up to date with the latest changes.

Reply:
Ok!

Reply:
> But for sure, tell me what you think should be there and I correct it. ;-) 

I would be in favor of dropping everything and keep only some meaningful overrides (codegen-units and lto for release and bench?). The compiler changes between rustc versions so I am afraid hardcoding the configuration is not enough for reproducibility between different rustc versions (and defaults stay the same if you keep the same rustc...).

Reply:
/done